From 4ff989fe7f4235d0621b16d406cce1b1933a2ae2 Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@resin.io>
Date: Mon, 12 Feb 2018 17:52:32 +0000
Subject: [PATCH 4/5] Avoid depending on systemd ask password path unit

We mask this service so no need to depend on it. We avoid a system warning in
this way too:

	systemd[1]: systemd-ask-password-plymouth.path: Cannot add dependency job,
	ignoring: Unit systemd-ask-password-plymouth.path is masked.

Upstream-status: Innappropiate [Configuration]
Signed-off-by: Andrei Gherzan <andrei@resin.io>
---
 systemd-units/plymouth-start.service.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/systemd-units/plymouth-start.service.in b/systemd-units/plymouth-start.service.in
index a82ee30..aaae979 100644
--- a/systemd-units/plymouth-start.service.in
+++ b/systemd-units/plymouth-start.service.in
@@ -1,8 +1,8 @@
 [Unit]
 Description=Show Plymouth Boot Screen
 DefaultDependencies=no
-Wants=systemd-ask-password-plymouth.path systemd-vconsole-setup.service systemd-udev-settle.service
-After=systemd-vconsole-setup.service systemd-udev-trigger.service systemd-udevd.service systemd-udev-settle.service resin-boot.service
+Wants=systemd-vconsole-setup.service systemd-udev-settle.service
+After=systemd-udev-trigger.service systemd-udevd.service systemd-udev-settle.service resin-boot.service
 Before=systemd-ask-password-plymouth.service
 Requires=resin-boot.service
 ConditionKernelCommandLine=!plymouth.enable=0
-- 
2.7.4

